rts/lra/lra-test/lra-test-arquillian-extension/src/main/java/io/narayana/lra/arquillian/spi/NarayanaLRARecovery.java

```diff
@@ -0,0 +1,67 @@
+package io.narayana.lra.arquillian.spi;
+
+import io.narayana.lra.LRAConstants;
+import io.narayana.lra.client.NarayanaLRAClient;
+import io.narayana.lra.logging.LRALogger;
+import org.eclipse.microprofile.lra.tck.service.spi.LraRecoveryService;
+
+import javax.enterprise.context.ApplicationScoped;
+import javax.ws.rs.client.Client;
+import javax.ws.rs.client.ClientBuilder;
+import javax.ws.rs.client.WebTarget;
+import javax.ws.rs.core.Response;
+import java.net.URI;
+
+@ApplicationScoped
+public class NarayanaLRARecovery implements LraRecoveryService {
```

Comment:
Can't we change the camel case to LRARecoveryService

Reply:
What is the decision here? It looks like you changed something but the class name is still  NarayanaLraRecovery

Reply:
I must have misunderstood you. LraRecoveryService is coming from spec, I will revert this back to NarayanaLRAREcovery then and change the spec later in a single sweep over all classes.

Reply:
Changing the spec might be more of a problem. But I would at least prefer our implementation to be consistent in how we interpret Camel Case.